tests/utils/kernel/inc/stl_kernel/kernel_test_constants.h

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// This header is intended to be force included into the .cpp under test.
+// KERNEL_TEST_NAME will then be provided as a /D switch and preprocessor
+// concatenated into one big string literal.

```

Comment:
This approach may need to change, depending on whether we can figure out another way to do multi-TU tests or not.